content-review/impl/turnitin-oc/src/main/java/org/sakaiproject/contentreview/turnitin/oc/ContentReviewServiceTurnitinOC.java

```diff
@@ -159,8 +166,26 @@
 	private static final String VIEWER_PERMISSION_MAY_VIEW_SUBMISSIONS_FULL_SOURCE = "may_view_submission_full_source";
 	private static final String VIEWER_PERMISSION_MAY_VIEW_MATCH_SUBMISSION_INFO = "may_view_match_submission_info";
 	private static final String VIEWER_DEFAULT_PERMISSIONS = "viewer_default_permission_set";
-	private static final String INSTRUCTOR = "INSTRUCTOR";
-	private static final String LEARNER = "LEARNER";
+	private enum ROLES{
+		INSTRUCTOR(Arrays.asList("Faculty", "Instructor", "Mentor", "Staff", "maintain", "Teaching Assistant")),
```

Comment:
I just looked through the existing default templates... are there any default Sakai roles that I may have missed? Custom roles can be mapped in the sakai.properties?